### PR TITLE
Phy update

### PR DESCRIPTION
**Pull request by SeppoTakalo**
Requires: https://github.com/ARMmbed/nanostack-border-router/pull/21

### CODE REVIEW

**Comment by SeppoTakalo** [comment]
@hasnainvirk Please review.

File: mbed_app.json [hunk 1]
```diff
@@ -27,11 +31,11 @@
...
 		"prefix-from-backhaul": true,
-		"rf-channel": 12,
-		"rf-channel-page": 0,
-		"rf-channel-mask": "0x07fff800",
+		"rf-channel": 10,
+		"rf-channel-page": 2,
+		"rf-channel-mask": "(1<<10)",
 		"short-mac-address": "0xface",
-		"backhaul-bootstrap-mode": "NET_IPV6_BOOTSTRAP_AUTONOMOUS",
+		"backhaul-bootstrap-mode": "NET_IPV6_BOOTSTRAP_STATIC",
```
Comment (hasnainvirk) [inline mbed_app.json]: Please don't change the mbed_app.json

**Comment by hasnainvirk** [comment]
https://github.com/ARMmbed/nanostack-border-router/pull/21/files This needs to be merged first in order to get the CI passed,

File: source/rf_wrapper.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include "rf_wrapper.h"
+
+// Must be defined for next preprocessor tests to work
+#define ATMEL       0
+#define MCR20       1
+#define NCS36510    2
+
+#if MBED_CONF_APP_RADIO_TYPE == ATMEL
+#include "NanostackRfPhyAtmel.h"
+NanostackRfPhyAtmel rf_phy(ATMEL_SPI_MOSI, ATMEL_SPI_MISO, ATMEL_SPI_SCLK, ATMEL_SPI_CS,
```
Comment (hasnainvirk) [inline source/rf_wrapper.cpp]: Can we enhance the code to check for Radio type too ? At least in case of ATMEL ? Just in case if someone wants to use SubG

**Pull request merged**